src/html.rs

```diff
@@ -40,6 +40,9 @@ struct Ctx<'b, I> {
     table_state: TableState,
     table_alignments: Vec<Alignment>,
     table_cell_index: usize,
+    // number of code blocks we're in. this may never become
```

Comment:
From what I can tell, changes won't be needed in this file, the unescaping logic should be done during parsing.